Tasks/AzureAppConfigurationSnapshotV1/task.json

```diff
@@ -2,14 +2,14 @@
     "id": "520c1ef0-be95-4931-9278-6e3ac79b81f2",
     "name": "AzureAppConfigurationSnapshot",
     "friendlyName": "Azure App Configuration Snapshot",
-    "description": "Create a snapshot in an Azure App Configuration instance",
+    "description": "Create a snapshot in an Azure App Configuration instance in your build and deployment pipelines",
```

Comment:
How does this sound to you?

Create a configuration snapshot in Azure App Configuration through build and deployment pipelines.